content/en/docs/tutorials/cluster-management/install-use-dra.md

```diff
@@ -0,0 +1,540 @@
+            nodeSelectorTerms:
+            - matchFields:
+              - key: metadata.name
+                operator: In
+                values:
+                - kind-worker
+        reservedFor:
+        - name: pod0
+          resource: pods
+          uid: fa55b59b-d28d-4f7d-9e5b-ef4c8476dff5
+    kind: List
+    metadata:
+      resourceVersion: ""
+    {{< /highlight >}}
+
+1.  Observe the driver by checking the pod logs for pods backing the driver
+   daemonset:
+
+    ```shell
+    kubectl logs -l app.kubernetes.io/name=dra-example-driver -n dra-tutorial
+    ```
+
+    The output is similar to this:
+    ```
+    I0729 05:11:52.679057       1 driver.go:84] NodePrepareResource is called: number of claims: 1
+    I0729 05:11:52.684450       1 driver.go:112] Returning newly prepared devices for claim '79e1e8d8-7e53-4362-aad1-eca97678339e': [&Device{RequestNames:[some-gpu],PoolName:kind-worker,DeviceName:gpu-4,CDIDeviceIDs:[k8s.gpu.example.com/gpu=common k8s.gpu.example.com/gpu=79e1e8d8-7e53-4362-aad1-eca97678339e-gpu-4],}]
+    ```
+
+You have now successfully deployed a Pod with a DRA based claim, and seen it
+scheduled to an appropriate node and the associated DRA APIs updated to reflect
+its status.
+
+## Remove the Pod with a claim
+
+When a Pod with a claim is deleted, the DRA driver deallocates the resource so
+it can be available for future scheduling. You can observe that by deleting our
+pod with a claim and seeing that the state of the ResourceClaim changes.
+
+###  Delete the pod using the resource claim
+
+1.  Delete the pod directly:
+
+    ```shell
+    kubectl delete pod pod0 -n dra-tutorial
+    ```
+
+    The output is similar to this:
+
+    ```
+    pod "pod0" deleted
+    ```
+
+### Observe the DRA state
+
+The driver will deallocate the hardware and update the corresponding
+ResourceClaim resource that previously held the association.
+
+1.  Check the ResourceClaim is now pending:
+
+    ```shell
+    kubectl get resourceclaims -n dra-tutorial
+    ```
+
+    The output is similar to this:
+    ```
+    NAME       STATE     AGE
+    some-gpu   pending   76s
+    ```
+
+1.  Observe the driver logs and see that it processed unpreparing the device for
+   this claim:
+
+    ```shell
+    kubectl logs -l app.kubernetes.io/name=dra-example-driver -n dra-tutorial
+    ```
+    The output is similar to this:
+    ```
+    I0729 05:13:02.144623       1 driver.go:117] NodeUnPrepareResource is called: number of claims: 1
+    ```
+
+You have now deleted a Pod that had a claim, and observed that the driver took
+action to unprepare the underlying hardware resource and update the DRA APIs to
+reflect that the resource is available again for future scheduling.
+
+## {{% heading "cleanup" %}}
+
+To cleanup the resources, delete the namespace for the tutorial which will clean up the ResourceClaims, driver components, and ServiceAccount. Then also delete the cluster level DeviceClass resource and cluster level RBAC resources.
```

Comment:
Shouldn't the driver clean up the DeviceClass when it's deleted?

Reply:
In the case of the example driver, it doesn't; the DeviceClass is in fact deployed (and undeployed) as part of the helm chart under normal operation. Tagging this [here in the scratch for forthcoming driver author documentation](https://docs.google.com/document/d/11Ki7CYwhBdd5ZRlnLRrvFD0G_chTDJShsaDS6i6VNWo/edit?tab=t.0#bookmark=id.9dizbhh2957t ). Some issues with letting the driver do that are:
* A given node local driver can't know if the DeviceClass is still representing drivers functioning on other nodes
* In seamless upgrade case, the downturning driver shouldn't remove the DeviceClass underneath the upcoming driver